scripts/_lines_of_code.py

```diff
@@ -64,9 +64,10 @@ def __init__(self, contributed_to: Repository, user):
             if commit.author is not None:
                 if commit.author.id == user.id:
                     for file in commit.files:
-                        if not file.filename in excepted_files or file.filename.startswith('node_modules/'):
-                            self.total_lines_of_addition_in_contribution += file.additions
-                            self.total_lines_of_addition_in_contribution += file.deletions
+                        if file.filename.startswith('node_modules') is False:
+                            if not file.filename in excepted_files:
+                                self.total_lines_of_addition_in_contribution += file.additions
+                                self.total_lines_of_deletion_in_contribution += file.deletions
```

Comment:
Make it more readable!
```suggestion
                            if not (file.filename in excepted_files or file.filename.startswith('node_modules')):
                                self.total_lines_of_addition_in_contribution += file.additions
                                self.total_lines_of_deletion_in_contribution += file.deletions
```